dojson/contrib/marc21/schemas/marc21/bibliographic/bdleader.json

```diff
@@ -0,0 +1,9 @@
+{
+    "type": "object",
+    "properties": {
+        "leader": {
+            "description": "Leader",
+            "type": "string"
```

Comment:
Can you elaborate on this? I basically just committed what I wrote in your office.